packages/themes/blue-jeans/theme/theme.scss

```diff
@@ -217,6 +217,16 @@
         lg: (width: calc(21.875rem * var(--ods-size-ratio))),
         xl: (width: calc(31.25rem * var(--ods-size-ratio))),
       ),
+      $switch: (
+        sm: (
+          height: var(--ods-size-8),
```

Comment:
In browser, height is 21px instead of 24px. In Chromium, it is saying the variable is not defined